lib/procore/requestable.rb

```diff
@@ -183,7 +188,7 @@ def sync(path, body: {}, options: {})
         batch_size: batch_size,
       )
 
-      groups = body[:updates].in_groups_of(batch_size, false)
+      groups = body[:updates].each_slice(batch_size).to_a
```

Comment:
This hopefully fixes a tricky bug causing a test failure in TravisCI on this line for Ruby versions 2.5 and 2.6:
```
NoMethodError: undefined method `in_groups_of' for #<Array:0x0000000003bec2a8>
```
https://travis-ci.org/github/procore/ruby-sdk/jobs/748880934

`in_groups_of` is a Rails method, so I replaced it with the Ruby equivalent `each_slice`.
https://stackoverflow.com/a/34859781/4383727